src/bugReport.js

```diff
@@ -68,12 +69,24 @@ function setLogSource(source) {
   logSource = source;
 }
 
+function disableDataUpload() {
+  dataUploadDisabled = true;
+}
+
+function isDataUploadDisabled() {
+  return dataUploadDisabled;
+}
+
 /**
  * _uploadBugScreenshot - convert annotated screenshot to image, then upload it
  *
  * @return {object} XHR Promise object for the image upload request
  */
 function _uploadBugScreenshot() {
+  if (dataUploadDisabled) {
+    return Promise.resolve();
```

Comment:
itt miért van Promise.resolve, amikor lentebb false van? A többi helyen is

Reply:
Javítva. Jó lesz a false, mert azt a hívó oldalon mindig filterezzük.